### PR TITLE
Organize Environment Variables better

### PR DESCRIPTION
**Pull request by aivantg**
## What's in this PR?

- Environment variables all come from common `.env` file
- No more .env.example files

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -8,6 +8,11 @@ const {
   generateSchemaFile
 } = require('../lib/generators');
 
+// Load regular config vars
+require('dotenv').config({
+  path: '.env'
```
Comment (annieyro) [inline bin/index.js]: Q: if developers already have various `.env` files (we have `.env.development` and `.env.production` files), they just need to create a new one, right? I wonder if this `path` can be an optional variable in the `settings` object from `package.json` 😓 sorry for nitpicking lol

File: .env.example [hunk 2]
```diff
@@ -1,2 +1,3 @@
+AIRTABLE_BASE_ID=
```
Comment (annieyro) [inline .env.example]: This file should be removed now right?

**Review by annieyro — COMMENTED** [review]
Wondering the path to the `.env` file can be specified by the user, but otherwise looks good!

File: bin/index.js (hunk 1 above)
Comment (aivantg) [reply bin/index.js]: ohh good idea! can do

File: .env.example (hunk 2 above)
Comment (aivantg) [reply .env.example]: yeah

**Pull request merged**